components/logins/src/sync/engine.rs

```diff
@@ -504,6 +506,22 @@ impl SyncEngine for LoginsSyncEngine {
     }
 }
 
+// This seems misplaced - but everywhere else a mirror row is read we don't need the unknown
+// fields because they are, um, unknown :) So we never merge or inspect them, just send
+// whatever we have in outgoing.
+fn unknown_fields_from_row(
+    row: &rusqlite::Row<'_>,
+) -> Result<Option<super::payload::UnknownFields>> {
+    Ok(match row.get::<_, Option<String>>("unknown_fields")? {
+        None => None,
```

Comment:
Maybe this should be an error?  I think it indicates that you want to get the `unknown_fields`, but forgot to include it in your SELECT list.